test.js

```diff
@@ -226,6 +226,19 @@ test('handles youtube attribution_links', t => {
 	t.is(fn('http://www.youtube.com/attribution_link?u=/watch?v=ABC12302&feature=share&list=UUsnCjinFcybOuyJU1NFOJmg&a=LjnCygXKl21WkJdyKu9O-w').service, 'youtube');
 });
 
+test('handles youtube google link', t => {
```

Comment:
Can we change these to use the word "redirection" so it's more clear to future contributors or those reading the tests?

So: `'handles youtube google link'` => `handles google redirection to youtube`